src/extractors/extractors/test/extractors/figma.test.ts

```diff
@@ -467,11 +471,11 @@ describe('Figma', () => {
           {
             name: 'Foobar Typograph',
             initializer:
-              'new Typograph({fontSize: 9001, color: Color.rgba(100, 100, 100, 1), font: helloFonts.Foobar.BoldItalic})',
+              'new Typograph({letterSpacing: 0.4824822, fontSize: 9001, lineHeight: 11, color: Color.rgba(100, 100, 100, 1), font: helloFonts.Foobar.BoldItalic})',
```

Comment:
these seem like magic numbers and are repeated.  do we always just inline these things?